packages/launcher/src/components/launch/Launch.tsx

```diff
@@ -60,6 +69,25 @@ export default function Launch(): ReactElement {
   const [open, setOpen] = React.useState(false);
   const [clicked, setClicked] = React.useState(false);
   const [activeContainer, setActiveContainer] = React.useState<Dockerode.Container | null>(null);
+  const [axonVersions, setAxonVersions] = React.useState([""]);
+  const [requested, setRequested] = React.useState(false);
+  const [axonVersion, setAxonVersion] = React.useState("edge");
+
+  const getVersions = async () => {
+    const ipcRenderer: IpcRenderer = window.require("electron").ipcRenderer;
+
+    ipcRenderer.on("axon-tags", (event, args: string[]) => {
+      setAxonVersions(args);
+      setAxonVersion(args[args.length - 1]);
```

Comment:
The implementation of axon-tags does not sort the output. We should sort the output up there if we want to always pick the last element. I think a good sort order would be:

1. latest
2. semantic sort with most recent first
3. edge